tools/tertiary-analysis/garnett/garnett_check_markers.xml

```diff
@@ -0,0 +1,38 @@
+<tool id="garnett_check_markers" name="Garnett - check markers" version="@TOOL_VERSION@+galaxy0" profile="@PROFILE@">
```

Comment:
Can you add a tool description please? and make sure that tool name and description read nicely together in the UI.